README.md

```diff
@@ -322,7 +322,7 @@ hey -n 10000 -q 10 -c 5 http://$(terraform output k8s_master_public_ip):31190
 You can monitor the autoscaler events with:
 
 ```bash
-$ kubectl --kubeconfig ./$(terraform output kubectl_config) describe hpa
+$ while sleep 5; do kubectl --kubeconfig ./$(terraform output kubectl_config) describe hpa; done
```

Comment:
I think `watch` is shorter. Can you change it to watch please?

Reply:
Sure.. I started with watch but changed it for some reason. Changed. Cheers!